st2common/st2common/util/output_schema.py

```diff
@@ -93,8 +94,10 @@ def validate_output(runner_schema, action_schema, result, status, output_key):
         status = action_constants.LIVEACTION_STATUS_FAILED
         # include the error message and traceback to try and provide some hints.
         result = {
-            "error": str(ex),
-            "message": "Error validating output. See error output for more details.",
+            "exit_code": FAILURE_EXIT_CODE,
+            "result": "None",
+            "stderr": str(ex),
+            "stdout": "",
```

Comment:
validation is not a subprocess, so using `exit_code`, `stderr`, and `stdout` feels a little odd here.
Can we adjust something in st2web to pick up the error message instead?